src/modules/BootSupportCheck.rb

```diff
@@ -191,6 +194,8 @@ def check_zipl_part
         return false
       end
 
```

Comment:
This should work. Isn't it?
```ruby
boot_fs = staging.filesystems.find { f| f.mountpoint == "/boot/zipl" }
boot_fs ||= staging.filesystems.find { f| f.mountpoint == "/boot" }
boot_fs ||= staging.filesystems.find { f| f.mountpoint == "/" }

# It would be better to improve Filesystems::Type#is? to support :ext, but this will also make it
if ![:ext2, :ext3, :ext4].include? boot_fs.type.to_sym
  add_new_problem(_("Missing ext partition for booting. Cannot install boot code."))
  return false
end
```